platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/jmh/java/com/swirlds/platform/benchmark/consensus/ConsensusBenchmark.java

```diff
@@ -71,24 +66,13 @@ public class ConsensusBenchmark {
     private Consensus consensus;
 
     @Setup
-    public void setup() {
+    public void setup() throws Exception {
         final ConsensusTestDefinition testDefinition = new ConsensusTestDefinition(
                 "Performance Test", numNodes, (l, i) -> WeightGenerators.balancedNodeWeights(i), numEvents);
         testDefinition.setSeed(seed);
         events = testDefinition.getNode1EventEmitter().emitEvents(numEvents);
 
-        final ConfigurationBuilder configurationBuilder = ConfigurationBuilder.create();
-        // FUTURE WORK: replace this with ConfigurationUtils.scanAndRegisterAllConfigTypes() after it merges
-        try (ScanResult result = new ClassGraph().enableAnnotationInfo().scan()) {
-            ClassInfoList classInfos = result.getClassesWithAnnotation(ConfigData.class.getName());
-            classInfos.forEach(classInfo -> {
-                Class<? extends Record> type = (Class<? extends Record>) classInfo.loadClass();
-                configurationBuilder.withConfigDataType(type);
-            });
-        }
-        final Configuration configuration = configurationBuilder.build();
-        ConfigurationHolder.getInstance().setConfiguration(configuration);
-
+        final Configuration configuration = DefaultConfiguration.buildBasicConfiguration();
```

Comment:
The previous TODO mentions using `scanAndRegisterAllConfigTypes`, but I don't understand the implications of that vs. `buildBasicConfiguration`

Could you clarify?